apiserver/facades/client/bundle/mock_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bundle_test
+
+import (
+	"github.com/juju/testing"
+
+	"github.com/juju/description"
+)
+
+type mockState struct {
+	testing.Stub
+	model mockModel
+}
+
+func (m *mockState) Export() (description.Model, error) {
+	m.MethodCall(m, "Export")
+	if err := m.NextErr(); err != nil {
+		return nil, err
+	}
+	return m.model.desc, nil
```

Comment:
the result here should be a model.Description with stuff added that can be checked